keyboards/the_royal/liminal/keymaps/default/keymap.c

```diff
@@ -0,0 +1,165 @@
+#include QMK_KEYBOARD_H
+
+enum layer_names {
+    _QWERTY,
+    _LOWER,
+    _RAISE,
+    _OTHER
+};
+
+#define PERMISSIVE_HOLD
+
+#define TAPPING_TERM 200
+
+#define TAPPING_TERM_PER_KEY
```

Comment:
these won't have any effect -- they'll need to go into a new `config.h` in the keymap directory